seekret/apitest/_createleaf.py

```diff
@@ -0,0 +1,19 @@
+def create_leaf(path: str, value):
```

Comment:
This doesnt support arrays. Should it?

Reply:
I removed the support for now, I think it's not required for now.
We can add support when the need arises.